ResourcesPage.xaml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8" ?>
+   <!-- 
+         * Author: Taylor Showalter
+         * Date: 10/15/2024
+         * Description: Screen for Resource links
+         * Bugs: None Known
+         * Reflection: Took time, alot of getting mad at not realizing
+         * I had to "use Lab2_Solution" at the top of app.xaml.cs to be able 
+         * to launch on my screen to be able to see it. Lots of questions to 
+         * either Alex (while his is not on my lab team, he was still willing
+         * to help me fix certain errors) or Chat GPT on how to implement things 
+         * or what a skeleton of the code should look like in order to get rid of 
+         * errors or how to implement things like shadows since it is different 
+         * from html and css which I do know.
+   --> 
+    
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Lab2_Solution.ResourcesPage"
+             Title="ResourcesPage">
+    <!-- Define resources that can be used within this page -->
+    <ContentPage.Resources>
+        <!-- Define an array of strings (URLs) with the key 'UrlList' -->
+        <x:Array x:Key="UrlList" Type="{x:Type x:String}">
+            <!-- Each x:String element represents a URL in the list -->
+            <x:String>https://wisconsindot.gov/Pages/Home.aspx</x:String>
```

Comment:
Do the links work? I can't test this yet until you make the other changes, but I hope they do. Also, can you show something more "pilot-friendly", rather than a URL, the title of the page that the URL takes you to?

Reply:
they are real URLs but they are non functional hyperlinks